src/Microsoft.ML.Transforms/NormalizerCatalog.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML
+{
+    /// <summary>
+    /// Extensions for normalizer operations.
+    /// </summary>
+    public static class NormalizationCatalog
+    {
+        /// <summary>
+        /// Normalize (rescale) the column according to the specified <paramref name="mode"/>.
+        /// </summary>
+        /// <param name="catalog">The transform catalog</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
+        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="mode">The <see cref="NormalizingEstimator.NormalizerMode"/> used to map the old values in the new scale. </param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[Normalize](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Normalizer.cs)]
+        /// ]]>
+        /// </format>
+        /// </example>
+        public static NormalizingEstimator Normalize(this TransformsCatalog catalog,
+           string outputColumnName, string inputColumnName = null,
+            NormalizingEstimator.NormalizerMode mode = NormalizingEstimator.NormalizerMode.MinMax)
+            => new NormalizingEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnName, inputColumnName ?? outputColumnName, mode);
+
+        /// <summary>
+        /// Normalize (rescale) several columns according to the specified <paramref name="mode"/>.
+        /// </summary>
+        /// <param name="catalog">The transform catalog</param>
+        /// <param name="mode">The <see cref="NormalizingEstimator.NormalizerMode"/> used to map the old values to the new ones. </param>
+        /// <param name="columns">The pairs of input and output columns.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[Normalize](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Normalizer.cs)]
+        /// ]]>
+        /// </format>
+        /// </example>
+        public static NormalizingEstimator Normalize(this TransformsCatalog catalog,
+            NormalizingEstimator.NormalizerMode mode,
```

Comment:
>NormalizerMode [](start = 33, length = 14)

NormalizationMode?

Reply:
Ok.

---
In reply to: [263234251](https://github.com/dotnet/machinelearning/pull/2865#discussion_r263234251) [](ancestors = 263234251)